localized/en-us/games/wordwindow/wordwindow_level.tsv

```diff
@@ -187,55 +187,55 @@ en-US	14	1	5	4	"बच्चे अपनी कक्षा को सजान
 en-US	16	2	1	3	"चरागाह में ३ भेड़ें हैं। प्रत्येक भेड़ के ४ बच्चे होते हैं। चरागाह में कितने मेमने हैं?"	¿Úæ»æã ×ð¢ 3 ÖðÇ¸ð¢ ãñ¢Ð ÂýˆØð·UUUU ÖðÇ¸ ·ðUUU 4 Õ‘¿ð ãôÌð ãñ¢Ð ¿Úæ»æã ×ð¢ ç·UUUUÌÙð ×ð×Ùð ãñ¢?	6	9	12	15	lambs	lambs	lambs	lambs	3 X 4 = <12>	3	3	4	0	0					wordwindow-stone-yellow.png	wordwindow-stone-green.png		
-en-US	16	2	2	3	"एक ब8ाख के २ पैर होते हैं। ३ बतख के कितने पैर हैं?"	°·UUUU Õ}æ¹ ·ðUUU 2 ÂñÚ ãôÌð ãñ¢Ð 3 ÕÌ¹ ·ðUUU ç·UUUUÌÙð ÂñÚ ãñ¢?	3	6	9	12	legs	legs	legs	legs	3 X 2 = <6>	2	3	2	0	0					wordwindow-stone-green.png	wordwindow-stone-red.png		
+en-US	16	2	2	3	"एक बतख के २ पैर होते हैं। ३ बतख के कितने पैर हैं?"	°·UUUU Õ}æ¹ ·ðUUU 2 ÂñÚ ãôÌð ãñ¢Ð 3 ÕÌ¹ ·ðUUU ç·UUUUÌÙð ÂñÚ ãñ¢?	3	6	9	12	legs	legs	legs	legs	3 X 2 = <6>	2	3	2	0	0					wordwindow-stone-green.png	wordwindow-stone-red.png		
 en-US	16	2	3	3	"खेत पर २ वैगन हैं। प्रत्येक वैगन में ८ बच्चे होते हैं। कितने बच्चे वैगनों में हैं?"	¹ðÌ ÂÚ 2 ßñ»Ù ãñ¢Ð ÂýˆØð·UUUU ßñ»Ù ×ð¢ 8 Õ‘¿ð ãôÌð ãñ¢Ð ç·UUUUÌÙð Õ‘¿ð ßñ»Ùô¢ ×ð¢ ãñ¢?	4	8	12	16	children	children	children	children	2 X 8 = <16>	4	2	8	0	0					wordwindow-stone-yellow.png	wordwindow-stone-blue.png		
 en-US	16	2	4	3	"किट, रूथ और डीन ने पिज्जा के २ स्लाइस खाए। पिज्जा के कितने स्लाइस उन्होंने सभी में खाए?"	ç·UUUUÅ, MUUUUÍ ¥æñÚ ÇèÙ Ùð çÂ”ææ ·ðUUU 2 SÜæ§â ¹æ°Ð çÂ”ææ ·ðUUU ç·UUUUÌÙð SÜæ§â ©‹ãô¢Ùð âÖè ×ð¢ ¹æ°?	3	6	12	15	slices	slices	slices	slices	3 X 2 = <6>	2	3	2	0	0					wordwindow-stone-blue.png	wordwindow-stone-yellow.png		
-en-US	16	2	5	3	"शेल्ली ने डोनट्स के ४ बॉ1स खरीदे। प्रत्येक बॉ1स में ५ डोनट्स होते हैं। उसने कुल कितने डोनट्स खरीदे?"	àæðËÜè Ùð ÇôÙÅ÷â ·ðUUU 4 Õæòvâ ¹ÚèÎðÐ ÂýˆØð·UUUU Õæòvâ ×ð¢ 5 ÇôÙÅ÷â ãôÌð ãñ¢Ð ©âÙð ·éUUUÜ ç·UUUUÌÙð ÇôÙÅ÷â ¹ÚèÎð?	5	15	20	30	donuts	donuts	donuts	donuts	4 X 5 = <20>	3	4	5	0	0					wordwindow-stone-red.png	wordwindow-stone-green.png		
-en-US	16	3	1	3	"माइक के पास ३ जोड़ी मोजे हैं। प्रत्येक जोड़ी में २ मोजे हैं। माइक के पास कितने मोज़े हैं?"	×æ§·UUUU ·ðUUU Âæâ 3 ÁôÇ¸è ×ôÁð ãñ¢Ð ÂýˆØð·UUUU ÁôÇ¸è ×ð¢ 2 ×ôÁð ãñ¢Ð ×æ§·UUUU ·ðUUU Âæâ ç·UUUUÌÙð ×ôÁ¸ð ãñ¢?	3	6	9	12	socks	socks	socks	socks	3 X 2 = <6>	2	3	2	0	0					wordwindow-stone-red.png	wordwindow-stone-yellow.png		
+en-US	16	2	5	3	"शेल्ली ने डोनट्स के ४ बॉक्स खरीदे। प्रत्येक बॉक्स में ५ डोनट्स होते हैं। उसने कुल कितने डोनट्स खरीदे?"	àæðËÜè Ùð ÇôÙÅ÷â ·ðUUU 4 Õæòvâ ¹ÚèÎðÐ ÂýˆØð·UUUU Õæòvâ ×ð¢ 5 ÇôÙÅ÷â ãôÌð ãñ¢Ð ©âÙð ·éUUUÜ ç·UUUUÌÙð ÇôÙÅ÷â ¹ÚèÎð?	5	15	20	30	donuts	donuts	donuts	donuts	4 X 5 = <20>	3	4	5	0	0					wordwindow-stone-red.png	wordwindow-stone-green.png		
+en-US	16	3	1	3	"माइक के पास ३ जोड़ी मोजे हैं। प्रत्येक जोड़ी में २ मोजे हैं। माइक के पास कितने मोजे हैं?"	×æ§·UUUU ·ðUUU Âæâ 3 ÁôÇ¸è ×ôÁð ãñ¢Ð ÂýˆØð·UUUU ÁôÇ¸è ×ð¢ 2 ×ôÁð ãñ¢Ð ×æ§·UUUU ·ðUUU Âæâ ç·UUUUÌÙð ×ôÁ¸ð ãñ¢?	3	6	9	12	socks	socks	socks	socks	3 X 2 = <6>	2	3	2	0	0					wordwindow-stone-red.png	wordwindow-stone-yellow.png		
```

Comment:
Update Chanakya text

Reply:
Chanakya correct for donuts  and updated for mike